go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/grpc-ecosystem/grpc-gateway/v2
 
-go 1.22.7
-
-toolchain go1.23.4
+go 1.22.0
```

Comment:
Thanks for this, we only updated the version because gRPC updated their version. I expect we won't be able to merge this until there's a new tagged gRPC version with the relaxed Go constraint.

Reply:
gRPC release a new version relaxing the version requirement https://github.com/grpc/grpc-go/releases/tag/v1.68.1. This was merged in https://github.com/grpc-ecosystem/grpc-gateway/pull/5018.